tests/__fixtures__/payments/payments.fixtures.ts

```diff
@@ -271,6 +271,18 @@ export const mockFetchPaymentMethodsResponse = {
       code: 'PayPalExp',
       paymentOptions: [],
     },
+    {
+      type: 'CustomerAccount',
+      id: 'cada88b2-aa03-460a-8483-c53f431ff9b0',
+      description: 'iDEAL',
+      code: 'iDEAL',
+      paymentOptions: [],
+      issuers: [
+        { id: '0805', name: 'RaboBank' },
```

Comment:
Give it a different id. This is used already on the next line

Reply:
Ahaha my mistake 😅
Fixed

Reply:
Thanks :)